gui/src/app/direct/directHeader.tpl.html

```diff
@@ -32,7 +32,7 @@
                     <div msg="" key="navbar.messageValidator">Message Validator</div>
                 </a>
             </li>
-            <li uib-dropdown ng-class="{active: $state.includes('direct.ccda') || $state.includes('direct.ccdar2') || $state.includes('direct.ccdar3')}">
+            <li uib-dropdown ng-class="{active: $state.includes('direct.ccda') || $state.includes('direct.ccdar2') || $state.includes('direct.ccdar3') || $state.includes('direct.svap})">

```

Comment:
-I think you are missing a single-quote after svap
-I think that the } and the ) need to be swapped
Also, I'm curious if the CSS class exists in the CSS? I haven't checked.